share-coordinator/src/main/java/org/apache/kafka/coordinator/share/ShareCoordinatorRecordSerde.java

```diff
@@ -29,9 +29,9 @@ public class ShareCoordinatorRecordSerde extends CoordinatorRecordSerde {
     @Override
     protected ApiMessage apiMessageKeyFor(short recordVersion) {
         switch (recordVersion) {
-            case ShareCoordinator.SHARE_SNAPSHOT_RECORD_KEY_VERSION:
+            case 0:
```

Comment:
This is a temporary step backward. I will reword it in https://issues.apache.org/jira/browse/KAFKA-18308.